app/models/version.rb

```diff
@@ -318,15 +318,16 @@ def to_title
     end
   end
 
-  def to_bundler
+  def to_bundler(locked_version: false)
+    modifier = (locked_version && prerelease?) ? "" : "~> "
```

Comment:
Maybe we can just make the prerelease block explicit, something like this (I don't think you'll be able to directly apply this suggestion, but just to give the idea of what it would look like).

```suggestion
    if prerelease?
      modifier = locked_version ? "" : "~> "
      %(gem '#{rubygem.name}', '#{modifier}#{number}')
    elsif number[0] == "0"
      %(gem '#{rubygem.name}', '~> #{number}')
```